lib/datahub-client/tests/client/datahub/test_search.py

```diff
@@ -123,6 +130,9 @@ def test_one_search_result(mock_graph, searcher):
                 metadata={
                     "owner": "",
                     "owner_email": "",
+                    "parent_container_display_name": "abc",
```

Comment:
Could this be passed back as an `EntityRef` object rather than adding to this big dictionary? It gives us a bit more type safety and this metadata dict feels like a bit of tech debt that is no longer helping very much.

EntityRef already has both `urn` and `display_name` attributes.

Reply:
We might need to change entity ref. If we end up having / collecting display names for databases and then their actual names we would want EntityRef to hold both. Can tackle that if it comes though i suppose